lib/site_prism/addressable_url_matcher.rb

```diff
@@ -72,29 +72,33 @@ def extract_component_templates
           component_url = component_url.sub(substituted_value, template_value)
         end
 
-        component_templates[component] = Addressable::Template.new(component_url.to_s)
+        component_templates[component] =
+          Addressable::Template.new(component_url.to_s)
       end
     end
 
-    # Returns empty hash if the template omits the component, a set of substitutions if the
-    # provided URI component matches the template component, or nil if the match fails.
+    # Returns empty hash if the template omits the component,
+    # a set of substitutions if the
+    # provided URI component matches the template component,
+    # or nil if the match fails.
     def component_matches(component, uri)
-      extracted_mappings = {}
-      component_template = component_templates[component]
-      if component_template
+      mappings = {}
+      if (component_template = component_templates[component])
```

Comment:
I personally dislike this kind of assignment, so I altered it to the previous one

Reply:
I was trying to leverage between line length and method length. Currently, we have 80 chars in 11 (code) lines

Reply:
Can we revert this one?